README.md

```diff
@@ -17,13 +17,20 @@ export function render () {
     { text: 'Tab 2', content: <div>Tab 2 content</div> },
     { text: 'Tab 3', content: <div>Tab 3 content</div> }
   ];
+  // optional onChange event
+  const onChange = ({ clickedIndex }) => {
+    console.log(`you clicked index ${clickedIndex}`)
+  };
 
-  return <TabControl active-tab-index={0} items={items} />;
+  return <TabControl activeTabIndex={0} items={items} onChange={onChange} removeOnHide={true} />;
 }
 
 ```
 
 ### Attributes
 
-#### active-tab-index
+#### activeTabIndex
 Set initial active tab index
+
+### removeOnHide
+If set to true, only the active tab content will be rendered.
```

Comment:
I think it's fine to have this here - for the current use case having this setting on would be great - but for other use cases it might be different. Getting the same classes & general DOM structure would be quite convenient.